docs/guidance/publishing-from-portal.md

```diff
@@ -1,9 +1,98 @@
+
+### (1) Make a commitment
+
+Consider making or advocating for a public commitment to publish standardised open data using the project level data specification and OCDS.
+
+Commitments are important to help align implementation with the goals of publishing open data and to help overcome technical, political or bureaucratic barriers to publication.
+
+Applications to join [CoST](http://infrastructuretransparency.org/) can be used to make a commitment or if your country is a member of the [Open Government Partnership](https://www.opengovpartnership.org/), your National Action Plan is another great place to start.
+
+Refer to the [OCDS implementation journey](https://www.open-contracting.org/implement/#/1) for information and resources about making commitments related to OCDS.
+
+### (2a) Map project level data and summary contracting process data
+
+Map existing data structures to the [project level data specification](../projects/index.md).
+
+```eval_rst
+.. admonition:: Tip
+    :class: Tip
+
+    .. markdown::
+
+      The [field level mapping template](https://docs.google.com/spreadsheets/d/1xHLf_w193pp97zfzhLc_LI-yEXrR_eyscga06Qo1blk/copy) can be used to document your mapping.
+
+```
+
+<!-- TODO: Update link to mapping template -->
+
+Your mapping might identify:
+
+* **Gaps in your data** where data in the project level data specification is not currently collected or disclosed in your system. Use the specification as a guide to the information that is important to users and consider whether your system and business processes could be updated to collect and publish additional information.
+
+* **Gaps in the data specification** where data is collected by your system but doesn’t map to the specification. Rather than being excluded from your publication, such information should be included as additional fields in your data. Refer to [extending the schema](../projects/schema.md#extending-the-schema) for information on including additional fields in your data.
+
+
+### (2b) Map detailed contracting process data
+
+If you collect detailed data on contracting processes, refer to the [OCDS implementation journey](https://www.open-contracting.org/implement/#/2) for information and resources about mapping and publishing your contracting data using OCDS.
+
+### (3) Build your data, systems and processes
+
+Create a project level JSON file for each project your system has information on and use a [JSON schema validator](https://json-schema.org/implementations.html#validators) to check that the files are structurally correct against the project level data specification.
+
+If you are also publishing contracting data using OCDS, create an OCDS release each time the data about a contracting process changes and use the [OCDS data review tool](http://standard.open-contracting.org/review/) to check your OCDS releases.
```

Comment:
Capitalize "Data Review Tool"

Reply:
Addressed in https://github.com/open-contracting/infrastructure/pull/46/commits/5c760acb02f31f25ee19ea29362dc314da9f633c